app/controllers/api/v2/users_controller.rb

```diff
@@ -0,0 +1,98 @@
+class Api::V2::UsersController < ApplicationController
+  include ApiHelper
+
+  def index
+    # TODO: ADD PERMISSIONS
+
+    res = User.find_by_sql "select * from users"
```

Comment:
same comment as above.

I suggest adding a "TODO" the way I did with all the other queries. I contend this is not a problem for v2 (because we are obsoleting it). We should do it in v3, though.